packages/localize/schematics/ng-add/index.ts

```diff
@@ -115,14 +114,16 @@ function moveToDependencies(host: Tree, context: SchematicContext) {
 
 export default function(options: Schema): Rule {
   return async (host: Tree) => {
-    if (!options.name) {
-      throw new SchematicsException('Option "name" is required.');
+    const projectName = options.name ?? options.project;
```

Comment:
nit: should we prefer using `options.project` and fallback to the deprecated `name` afterwards?
```suggestion
    const projectName = options.project ?? options.name;
```

Reply:
No because project can be populated even when not specified by the user. This is because it has a smart default.

Reply:
Ok, got it, thanks. Could you please add it as a comment in the code (so that this order is not treated as a problem)?

Reply:
Added